datalake_api/querier.py

```diff
@@ -196,7 +210,8 @@ def query_by_time(self, start, end, what, where=None, cursor=None):
             buckets = buckets[i:]
 
         for b in buckets:
-            kwargs = self._prepare_time_bucket_kwargs(b, what)
+            kwargs = self._prepare_time_bucket_kwargs(b, what,
+                                                      limit=MAX_RESULTS/2)
```

Comment:
Here's the comment explaining that choice:
https://github.com/planetlabs/datalake-api/blob/master/datalake_api/querier.py#L46